crates/ln-dlc-node/src/node/oracle_client.rs

```diff
@@ -4,11 +4,35 @@ use anyhow::Result;
 use bitcoin::secp256k1::XOnlyPublicKey;
 use dlc_manager::Oracle;
 use p2pd_oracle_client::P2PDOracleClient;
+use std::thread::sleep;
```

Comment:
🤔 do we have issues using std::thread instead of tokio?

Reply:
It's used in a blocking thread. Where we are using we actually cannot use `tokio::time::sleep`, because it's just an `FnOnce` closure, not a future.